packages/analytics-types/src/config.ts

```diff
@@ -77,16 +77,26 @@ export interface ReactNativeTrackingOptions extends TrackingOptions {
 
 export interface AdditionalBrowserOptions {
   attribution?: AttributionBrowserOptions;
+  trackPageViews?: boolean | PageTrackingBrowserOptions;
```

Comment:
Would this add it to the existing conifg param in init which we are trying to avoid?

Reply:
I was thinking of adding all types in one place. But it makes sense to host this part in the plugin.

Reply:
I was thinking we add them to the config only in the `analytics-ma-browser` package. See other comment about types.

Reply:
i think we should add the plugin specific types to the plugins package and the wrapper package can import it from there

Reply:
also curious about the decision to make this `boolean | PageTrackingBrowserOptions` how does that work?

Reply:
Removed it from typings. The boolean option will be handled in `analytics-ma-browser`. The options value is optional in the page view tracking plugin. Adding plugin means enabling the functionality.